frontend/notification/notifications_page.html

```diff
@@ -25,7 +25,7 @@ <h1>
                             <img ng-src="{{notification.from.photo_url}}" class="md-avatar" style="width: 60px; height: 60px;"/>
                             <div class="md-list-item-text" layout="column">
                                 <p><b style="font-size: 18px;">{{ notification.from.name }}</b></p>
-                                <p><b style="font-size: 15px; color: #009688;">{{ notification.current_institution.name }}</b></p>
+                                <p><b style="font-size: 15px; color: #009688;">{{ notification.current_institution.name || notification.from.institution_name }}</b></p>
```

Comment:
insert a comment explaining what is new notification data and what is old notification data